buildenv/jenkins/common/pipeline-functions

```diff
@@ -28,29 +28,46 @@
     // if not set, sniff the remote repositories references
 
     stage ('Sniff Repos') {
-        dir('openjdk') {
+        def description = ''
+
+        if (OPENJDK_SHA instanceof Map && OPENJDK_REPO instanceof Map && OPENJDK_BRANCH instanceof Map) {
```

Comment:
OPENJ9_SHA, and OMR_SHA also aren't passed as arguments. The mixing has not been introduced now.
The if statement is preserving the original implementation (on the else branch).
The purpose of passing a map instead of string is to keep the method signature unchanged.